ui/app/components/das/recommendation-card.hbs

```diff
@@ -2,7 +2,7 @@
   <section class="das-interstitial" style={{this.interstitialStyle}}>
     {{component (concat 'das/' this.interstitialComponent) proceed=this.proceedPromiseResolve error=this.error}}
   </section>
-{{else}}
+{{else if @summary.taskGroup}}
```

Comment:
This prevents a broken card from rendering when the final summary has been processed and the route is refreshing to ultimately display the empty message.